pkg/controller/controller_utils.go

```diff
@@ -183,15 +206,16 @@ type Expectations interface {
 
 // ControlleeExpectations track controllee creates/deletes.
 type ControlleeExpectations struct {
-	add int64
-	del int64
-	key string
+	add       int64
+	del       int64
+	key       string
+	timestamp time.Time
 }
 
-// Seen decrements the add and del counters.
-func (e *ControlleeExpectations) Seen(add, del int64) {
-	atomic.AddInt64(&e.add, -add)
-	atomic.AddInt64(&e.del, -del)
+// Add increments the add and del counters.
```

Comment:
This is much more readable and easier to comprehend than the `Seen()` language. Thanks!